### ISSUE TITLE
Not really a issue just a FYI

### ISSUE THREAD

**Issue by Artfulldodger** [IrcDirk/Carbonite-Classic#187]
Carbonite-Classic v1.02 for WOW Classic does not work since today's (9/29/21) wow update. 
But installing the WOW Burning Crusade Classic v2.0.8 in Wow classic seems to works just fine.

**Comment by IrcDirk** [comment]
Im not maintaining Carbonite-Classic only TBC version.

**Comment by Artfulldodger** [comment]
> 
> 
> Im not maintaining **Carbonite Classic** only **Carbonite Classic TBC** version.

Yes I already knew that thanks :) that's why I posted saying it's just a FYI. For others that may still be running the Classic version that doesn't work so they can get the TBC version and it will work in Classic as well. To me this is one of the Best addons I have as I use it for all 3 versions "Retail, TBC, and Classic" Thanks again for making this addon. <3

**Comment by AndehX** [comment]
Can confirm Carbonite no longer works with WoW Classic as of 1.14.4.  Not even v2.0.8 works anymore.

**Comment by IrcDirk** [comment]
Thanks for checking and letting everyone know.
